docker/build-tools/Dockerfile

```diff
@@ -60,7 +60,7 @@ ENV GO_JUNIT_REPORT_VERSION=df0ed838addb0fa189c4d76ad4657f6007a5811c
 ENV HADOLINT_VERSION=v2.12.0
 ENV HELM3_VERSION=v3.17.2
 # Please consult the docs team before bumping Hugo.
-ENV HUGO_VERSION=0.145.0
+ENV HUGO_VERSION=0.147.2
```

Comment:
```suggestion
ENV HUGO_VERSION=0.147.4
```

There's one every week, we have to cut off somewhere, but we may as well be up to date when they land